lib/grunt/deprecate.js

```diff
@@ -0,0 +1,116 @@
+var grunt = require('../grunt');
+
+function deprecate(obj, property, message) {
+  if (Array.isArray(obj)) {
```

Comment:
Instead of this "if array" logic, I'd just call `.forEach` on the array of to-be-deprecated items on line 38.